project_milestone_status/tests/common.py

```
@@ -0,0 +1,39 @@
+from odoo.tests import new_test_user
+from odoo.tests.common import TransactionCase
+
+
+class ProjectMilestoneStatusCommon(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.8.2)

Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)